assets/sass/components/global/_googlesitekit-change-badge.scss

```diff
@@ -26,9 +26,14 @@
 		font-size: $fs-label-sm;
 		font-weight: 500;
 		letter-spacing: $ls-xs;
-		line-height: $lh-label-lg;
+		line-height: $lh-title-lg;
```

Comment:
This should be `$lh-label-sm`, to align with the `label/small` token in Figma and the use of `font-size: $fs-label-sm` above. It's currently the same value as `$lh-title-lg`, but this could change in future.

```suggestion
		line-height: $lh-label-sm;
```

![image](https://github.com/google/site-kit-wp/assets/18395600/9586ffe2-0a21-4293-9ae3-b984ec33d21d)